### PR TITLE
docs: `SHOW FLOW` & `FLOWS` table info

### PR DESCRIPTION
**Pull request by discord9**
## What's Changed in this PR

- added flows table intro in information_schema
- added example of `SHOW FLOWS`

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [x] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/nightly/en/reference/sql/information-schema/flows.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# FLOWS
+The Flows table provide the flow task's information.
```
Comment (nicecui) [inline docs/nightly/en/reference/sql/information-schema/flows.md:2]:
```suggestion

The `Flows` table provides the flow task information.
```

File: docs/nightly/en/reference/sql/information-schema/flows.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ table_catalog    | String |     | NO   |         | FIELD
+ flow_definition  | String |     | NO   |         | FIELD
+ comment          | String |     | YES  |         | FIELD
+ expire_after     | Int64  |     | YES  |         | FIELD
+ source_table_ids | String |     | YES  |         | FIELD
+ sink_table_name  | String |     | NO   |         | FIELD
+ flownode_ids     | String |     | YES  |         | FIELD
+ options          | String |     | YES  |         | FIELD
+(10 rows)
+```
```
Comment (nicecui) [inline docs/nightly/en/reference/sql/information-schema/flows.md:19]:
```sql
public=> DESC TABLE INFORMATION_SCHEMA.FLOWS;
```

```sql
      Column      |  Type  | Key | Null | Default | Semantic Type 
------------------+--------+-----+------+---------+---------------
 flow_name        | String |     | NO   |         | FIELD
 flow_id          | UInt32 |     | NO   |         | FIELD
 table_catalog    | String |     | NO   |         | FIELD
 flow_definition  | String |     | NO   |         | FIELD
 comment          | String |     | YES  |         | FIELD
 expire_after     | Int64  |     | YES  |         | FIELD
 source_table_ids | String |     | YES  |         | FIELD
 sink_table_name  | String |     | NO   |         | FIELD
 flownode_ids     | String |     | YES  |         | FIELD
 options          | String |     | YES  |         | FIELD
(10 rows)
```

File: docs/nightly/en/reference/sql/information-schema/flows.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+The columns in table:
+
+* `flow_name`: the flow task's name.
+* `flow_id`: the flow task's id.
+* `table_catalog`: the catalog this flow belongs to, named as `table_catalog` to keep consistent with the `INFORMATION_SCHEMA` standard.
+* `flow_definition`: the flow task's definition. It's the SQL statement that were used to creates the flow task.
```
Comment (nicecui) [inline docs/nightly/en/reference/sql/information-schema/flows.md:26]:
```suggestion
* `flow_definition`: the flow task's definition. It's the SQL statement used to create the flow task.
```

File: docs/nightly/en/reference/sql/show.md [hunk 4]
```diff
@@ -176,3 +176,39 @@ WITH(
...
+  
+```sql
+public=> SHOW CREATE FLOW filter_numbers;
+      Flow      |                      Create Flow                      
+----------------+-------------------------------------------------------
+ filter_numbers | CREATE OR REPLACE FLOW IF NOT EXISTS filter_numbers  +
+                | SINK TO out_num_cnt                                  +
+                | AS SELECT number FROM numbers_input WHERE number > 10
+(1 row)
+```
```
Comment (nicecui) [inline docs/nightly/en/reference/sql/show.md:194]:
```sql
public=> SHOW CREATE FLOW filter_numbers;
```

```sql
      Flow      |                      Create Flow                      
----------------+-------------------------------------------------------
 filter_numbers | CREATE OR REPLACE FLOW IF NOT EXISTS filter_numbers  +
                | SINK TO out_num_cnt                                  +
                | AS SELECT number FROM numbers_input WHERE number > 10
(1 row)
```

File: docs/nightly/en/reference/sql/show.md [hunk 5]
```diff
@@ -176,3 +176,39 @@ WITH(
...
+
+Show all flows:
+
+```sql
+public=> SHOW FLOWS;
+     Flows      
+----------------
+ filter_numbers
+(1 row)
+```
```
Comment (nicecui) [inline docs/nightly/en/reference/sql/show.md:206]:
```sql
public=> SHOW FLOWS;
```

```sql
     Flows      
----------------
 filter_numbers
(1 row)
```

File: docs/nightly/en/reference/sql/show.md [hunk 6]
```diff
@@ -176,3 +176,39 @@ WITH(
...
+also support `LIKE` expression:
+``sql
+public=> show flows like "filter%";
+     Flows      
+----------------
+ filter_numbers
+(1 row)
+```
```
Comment (nicecui) [inline docs/nightly/en/reference/sql/show.md:214]:
```sql
public=> show flows like "filter%";
```

```sql
     Flows      
----------------
 filter_numbers
(1 row)
```

**Review by nicecui — COMMENTED** [review]
please also update the zh docs, thanks

File: docs/nightly/zh/reference/sql/information-schema/flows.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+ flownode_ids     | String |     | YES  |         | FIELD
+ options          | String |     | YES  |         | FIELD
+(10 rows)
+```
+
+表中的列：
+
+* `flow_name`: Flow 任务的名称。
+* `flow_id`: Flow 任务的 id。
+* `table_catalog`: the catalog this flow belongs to, named as `table_catalog` to keep consistent with the `INFORMATION_SCHEMA` standard. 该 Flow 所属的目录，命名为 `table_catalog` 以保持与 `INFORMATION_SCHEMA` 标准的一致性。
```
Comment (nicecui) [inline docs/nightly/zh/reference/sql/information-schema/flows.md:28]:
```suggestion
* `table_catalog`: 该 Flow 所属的目录，命名为 `table_catalog` 以保持与 `INFORMATION_SCHEMA` 标准一致。
```

File: docs/nightly/zh/reference/sql/information-schema/flows.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+* `flow_definition`: Flow 任务的定义。这是用于创建 Flow 任务的 SQL 语句。
```
Comment (nicecui) [inline docs/nightly/zh/reference/sql/information-schema/flows.md:29]:
```suggestion
* `flow_definition`: Flow 任务的定义，是用于创建 Flow 任务的 SQL 语句。
```

File: docs/nightly/zh/reference/sql/show.md [hunk 9]
```diff
@@ -176,3 +176,48 @@ WITH(
...
+public=> SHOW FLOWS;
+```
+
+```sql
+     Flows      
+----------------
+ filter_numbers
+(1 row)
+```
+also support `LIKE` expression:
```
Comment (nicecui) [inline docs/nightly/zh/reference/sql/show.md:213]:
```suggestion

同样也支持 `LIKE` 表达式：
```

**Review by nicecui — APPROVED** [review]

**Review by tisonkun — APPROVED** [review]

**Pull request merged**